### PR TITLE
add command for each function

### PR DESCRIPTION
**Pull request by greeschenko**
Hello,  I try to write vim9 plugins only vim config and I really like your plugin.
This pull request make small changes that have the following purpose:
1. allows not to use import in .vimrc
2. allows to use command line commands for not frequency used function and not to use key bind for it or if need  use function with parameters. As example you can use :FzfGrep Foo if keyword "Foo" not exist in your buffer

### CODE REVIEW

**Review by girishji — COMMENTED** [review]
See comments.

Update README and doc/* also.

**Comment by girishji** [comment]
Maybe it is time to implement this suggestion also: https://github.com/girishji/scope.vim/issues/11#issuecomment-2011713900

File: autoload/scope/fuzzy.vim [hunk 1]
```diff
@@ -74,7 +74,7 @@ var prev_grep = null_string
 # of parsing color codes, Vim's syntax highlighting is used. As Vim lacks
 # awareness of `grep`'s ignore-case flag, explicit instruction is needed for
 # accurate highlighting.
-export def Grep(grepCmd: string = null_string, ignorecase: bool = true, cword: string = null_string)
+export def Grep(cword: string = null_string, grepCmd: string = null_string, ignorecase: bool = true)
```
[the remark this answers is not in the document]
Comment (greeschenko) [reply autoload/scope/fuzzy.vim:77]: ok I do

File: plugin/scope.vim [hunk 2]
```diff
@@ -4,8 +4,35 @@ if !has('vim9script') ||  v:version < 901
...
+command!          FzfLoclistHistory    call fuzzy.LoclistHistory()	            #Entries in the location list stack
+command!          FzfMRU               call fuzzy.MRU()	                        #:h v:oldfiles
+command!          FzfMark              call fuzzy.Mark()	                    #Vim marks (:h mark-motions)
+command!          FzfOption            call fuzzy.Option()	                    #Vim options and their values
+command!          FzfQuickfix          call fuzzy.Quickfix()	                #Items in the quickfix list (sets 'current entry')
+command!          FzfQuickfixHistory   call fuzzy.QuickfixHistory()	            #Entries in the quickfix list stack
+command!          FzfRegister          call fuzzy.Register()	                #Vim registers, paste contents on <cr>
+command!          FzfTag               call fuzzy.Tag()	                        #:h ctags search
+command!          FzfWindow            call fuzzy.Window()	                    #Open windows
+
```
[the remark this answers is not in the document]
Comment (greeschenko) [reply plugin/scope.vim:38]: 1. ok I remove comments and maybe add some explanations in README
2. ok I change prefix from Fzf to Scope
3. I add wrapper function for Grep and yes it cat accept many arguments but I dont find how add boolean variable to the prompt command. Prompt read it as string

**Comment by greeschenko** [comment]
I not really understand what the point of this changes 
we already can use 
`nnoremap <yourkey> :call scope#fuzzy#Keymap()<CR>`
or
`nnoremap <yourkey> :vim9cmd call scope#fuzzy#Keymap()<CR>`

**Comment by girishji** [comment]
> I not really understand what the point of this changes we already can use `nnoremap <yourkey> :call scope#fuzzy#Keymap()<CR>` or `nnoremap <yourkey> :vim9cmd call scope#fuzzy#Keymap()<CR>`

Ignore.

File: plugin/scope.vim (hunk 2 above)
[the remark this answers is not in the document]
Comment (girishji) [reply plugin/scope.vim:38]: I will take care of adding arguments, and the documentation. Can you do it like this:
```
if !has('vim9script') ||  v:version < 901
    echoerr 'Needs Vim version 9.1 and above'
    finish
endif
vim9script

g:loaded_scope = true

import autoload '../autoload/scope/fuzzy.vim'

# ripgrep output can include column number. default values of grepformat does
# not contain column pattern, add it.
set grepformat^=%f:%l:%c:%m

var cmds = [
    'Autocmd',
    'BufSearch',
    'Buffer',
    'CmdHistory',
    'Colorscheme',
    'Command',
    'File',
    'Filetype',
    'GitFile',
    'Grep',
    'Help',
    'Highlight',
    'Keymap',
    'LspDocumentSymbol',
    'Loclist',
    'LoclistHistory',
    'MRU',
    'Mark',
    'Option',
    'Quickfix',
    'QuickfixHistory',
    'Register',
    'Tag',
    'Window',
]

def DoCommand(cmd: string)
    $'fuzzy.{cmd}()'->eval()
enddef

def Completer(prefix: string, line: string, cursorpos: number): string
    var parts = line->split()
    if parts[1] =~ $'^{prefix}'
        return cmds->copy()->filter((_, v) => v =~? $'^{prefix}')->join("\n")
    endif
    return null_string
enddef

command -nargs=+ -complete=custom,Completer Scope DoCommand(<f-args>)
```

**Review by girishji — COMMENTED** [review]
See my comment.

File: plugin/scope.vim (hunk 2 above)
[the remark this answers is not in the document]
Comment (greeschenko) [reply plugin/scope.vim:38]: OK I try this

**Comment by girishji** [comment]
https://github.com/girishji/scope.vim/commit/f740ead7c71cedf1d4eb4eb3050355252e7959e6

Now you can grep from command line

**Pull request merged**